client/src/components/Ticketing/ticketingmanager/showings/ShowingUpdated/EventShowingContainer.tsx

```diff
@@ -0,0 +1,64 @@
+import {Showing} from '../../../../../interfaces/showing.interface';
+import React, {useState} from 'react';
+import {EventShowingForm} from './EventShowingForm';
+import {EventShowingView} from './EventShowingView';
+import {useEvent} from './EventProvider';
+import {createDeleteFunction, createSubmitFunction} from './ShowingUtils';
+
+interface EventShowingContainerProps {
+  showing: Showing;
+}
+
+export const EventShowingContainer = (props: EventShowingContainerProps) => {
+  const {showing} = props;
+  const {setReloadShowing, token, setEditing, setPopUpProps} = useEvent();
+  const [edit, setEdit] = useState(false);
+
+  const onSuccess = async (event) => {
```

Comment:
`event` argument not utilized, it can be removed.

Reply:
Updated